src/api/app/models/project.rb

```diff
@@ -58,7 +58,10 @@ def autocomplete(search)
   has_many :messages, :as => :db_object, :dependent => :delete_all
   has_many :watched_projects, :dependent => :destroy, inverse_of: :project
 
-  has_many :linkedprojects, -> { order(:position) }, :class_name => 'LinkedProject', foreign_key: :db_project_id, :dependent => :delete_all
+  has_many :linked_projects, -> { order(:position) }, foreign_key: :db_project_id, :dependent => :delete_all
+  has_many :projects_linked, through: :linked_projects, class_name: 'Project', source: :linked_db_project
+  has_many :linking_projects, -> { order(:position) }, :class_name => 'LinkedProject', foreign_key: :linked_db_project_id, :dependent => :delete_all
+  has_many :projects_linking, through: :linking_projects, class_name: 'Project', source: :project
```

Comment:
@bgeuken, @hennevogel  What do you think about the naming? 
Finally I don't want to refactor the whole linking thing right now, so just name those relationships properly...

My proposal is naming it from the POV of the actual project:
```ruby
has_many :linking_to # I'm linking to other projects
has_many :projects_linking_to, through: linking_to # Projects that I'm linking to
has_many :linked_by # I'm linked by other projects
has_many :linked_by_projects, through: linked_by # Projects I'm linked by
```